frontend/src/features/properties/selector/MapSelectorContainer.tsx

```diff
@@ -42,7 +43,17 @@ export const MapSelectorContainer: React.FunctionComponent<IMapSelectorContainer
       <PropertySelectorTabsView
         activeTab={activeSelectorTab}
         setActiveTab={setActiveSelectorTab}
-        MapSelectorView={<PropertyMapSelectorFormView onSelectedProperty={onSelectedProperty} />}
+        MapSelectorView={
+          <PropertyMapSelectorFormView
+            onSelectedProperty={(property: IMapProperty) =>
```

Comment:
neither the search based property selector or the map based property selector should allow a duplicate to be selected.